src/status_im/chat/models.cljs

```diff
@@ -252,23 +258,24 @@
 (fx/defn navigate-to-chat
   "Takes coeffects map and chat-id, returns effects necessary for navigation and preloading data"
   [cofx chat-id {:keys [modal? navigation-reset?]}]
-  (cond
-    modal?
-    (fx/merge cofx
-              (navigation/navigate-to-cofx :chat-modal {})
-              (preload-chat-data chat-id))
+  (when (= (is-active? (get-chat cofx chat-id)) true)
```

Comment:
No need to `(= .. true)`  `(when (is-active? ...)` should do, unless you really mean to check it against true (i.e  `is-active?` might return some truthy values that you don't want to consider, but that does not seem to be the case).

Reply:
Thanks for the suggestion I will update this. I just wanted to say that I have not been able to receive notifications unless the person is already a contact in both versions. Is this intended?

Reply:
yes, so to receive a notification from `A` when `A` sends you a message, you need first to add `A` to your contacts